### ISSUE TITLE
Splitting advanced tab (discussion)

### ISSUE THREAD

**Issue by ToughGuyKunio** [SimonBrazell/privacy-redirect#125]
For me the advanced tab in v1.1.43 seems a bit cramped/messy. One suggestion would be to separate the advanced tab section into three different tabs, one tab per service: "Invidious", "Nitter" & "Bibliogram". 
This makes it easy to find what you are looking for. The boxes for entering instances could also be made larger, making changing instances a bit easier. Another benefit is that it leaves plenty of space if other options are added in the future.

**Comment by ToughGuyKunio** [comment]
Another suggestion that I think would be helpful is adding descriptions and examples. For example what "proactively remove twitter service worker" does and what "RegExp" is and an example of RegExp usage. Could either be a text under or a mouse-over feature.

**Comment by SimonBrazell** [comment]
Yeah was thinking the same too, that's why I added sub-headings in v1.1.43 but still pretty cluttered. 

The reason for the current limited space is that we are using the browser preferences page, will have to move it to a separate tab instead to give us more options / space.

**Comment by ToughGuyKunio** [comment]
Another one: I was thinking that it might be better or more future proof to rename "Invidious redirect" to "YouTube redirect", "Nitter Redirect" to "Twitter redirect", etc, in the settings.

It make sense since there are several supported alternatives/frontends for YouTube, Reddit and Google Search. In the future there might be other frontends for the other services as well. 

Perhaps "Nitter instance" could be renamed to "Twitter redirected to"

**Comment by SimonBrazell** [comment]
> Another one: I was thinking that it might be better or more future proof to rename "Invidious redirect" to "YouTube redirect", "Nitter Redirect" to "Twitter redirect", etc, in the settings.

Yeah probably a good idea, cheers!

**Comment by SimonBrazell** [comment]
I did a slight re-design of the `Advanced` tab in the latest version, using collapsible content for each provider instead, let me know what you think.

<img width="426" alt="Screen Shot 2021-01-11 at 14 18 29" src="https://user-images.githubusercontent.com/16571951/104144633-1a2c6f00-5418-11eb-9b54-5e43cc207731.png">
